documentation/tutorials/StateAssimilation/TreeRingSDA.Rmd

```diff
@@ -40,48 +40,66 @@ Perform a site-level SIPNET run using the following settings
 * At the top of the file add the following tags to set the ensemble size
 
 ```
-<state.data.assimilation>
-  <n.ensemble>35</n.ensemble>
-  <process.variance>FALSE</process.variance>
-  <sample.parameters>TRUE</sample.parameters>
+   <state.data.assimilation>
+   <n.ensemble>35</n.ensemble>
+   <process.variance>FALSE</process.variance>
+   <sample.parameters>TRUE</sample.parameters>
+   <data>
+    <format_id>
+    <format_id>1000000040</format_id>
+    </format_id>
+    <input.id>
+          <input.id>1000013298</input.id>
+    </input.id>
```

Comment:
Likewise, definitely shouldn't need to nest input.id within input.id. Also, should make sure we're consistent in naming tags between here an other parts of the code where input id's are specified.